apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-basic.tsx

```diff
@@ -146,7 +149,8 @@ export const AddAPIResourceBasic: FunctionComponent<AddAPIResourceBasicInterface
                         <Hint className="mb-0">
                             <Trans 
                                 i18nKey= { "extensions:develop.apiResource.wizard.addApiResource.steps.basic." +
-                                    "form.fields.identifier.hint" }>
+                                    "form.fields.identifier.hint" }
+                                tOptions={ { productName: productName } }>
```

Comment:
In L155, there is a reference to the cloud offering. Shall we update it to "WSO2 Identity Server"?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4369/commits/5f9e86491ef0a3af12f7235bb143cadc4347c5b8